docs/docsearch-scraper-config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "index_name": "dev_docs",
+  "start_urls": [
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/",
+      "selectors_key": "folder",
+      "page_rank": 0,
+      "tags": [
+        "folder"
+      ]
+    },
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/.+$",
+      "selectors_key": "folder",
+      "page_rank": 0,
+      "tags": [
+        "folder"
+      ]
```

Comment:
This tells the scraper, "Whenever the URL matches this line, we know it's a folder view in GitHub, so use the `folder` `selectors_key`."

We're also setting `page_rank` to 0 in an attempt to push these search results to the bottom. The trouble is that we can't separate out the search and indexing functions of the scraper, so we're adding all the GH tree view links to the search results. :disappointed: To account for that, we're pushing those results to the bottom.